projects/ui/src/lib/components/po-field/po-multiselect/po-multiselect-filter.service.ts

```diff
@@ -0,0 +1,58 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { PoMultiSelectFilter } from './interfaces/po-multiselect-filter.interface';
+import { PoMultiSelectOption } from './interfaces/po-multiselect-options.interface';
+
+@Injectable()
+export class PoMultiselectFilterService implements PoMultiSelectFilter {
+  fieldLabel: string = 'label';
+  fieldValue: string = 'value';
+
+  private _url: string;
+  private messages = [];
+
+  get url(): string {
+    return this._url;
+  }
+
+  constructor(private http: HttpClient) {}
+
+  getFilteredData({ value }: any): Observable<Array<PoMultiSelectOption>> {
+    const params = value ? { filter: value } : {};
+    return this.http
+      .get(`${this.url}`, {
```

Comment:
Poderia ser passado apenas `.get(this.url)` visto que esse cara já é uma string.